doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,935 @@
+
+  * Removed:
+
+    * An unused parameter of the :c:func:`nrf_cloud_connect` function.
+    * An unused :c:func:`nrf_cloud_shadow_update` function.
+
+* :ref:`lib_lwm2m_client_utils` library:
+
+  * Added support for using X509 certificates.
+
+* :ref:`lib_fota_download` library:
+
+  * Added an error code :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_INTERNAL` to indicate that the source of error is not network related.
+
+* :ref:`lib_nrf_cloud_rest` library:
+
+  * Updated by replacing the ``nrf_cloud_rest_cell_pos_get()`` function with :c:func:`nrf_cloud_rest_location_get`.
+
+* :ref:`lib_nrf_cloud_pgps` library:
+
+  * Added access to P-GPS predictions in external flash.
+
+  * Fixed:
+
+    * An issue where zero predictions would be requested from the cloud.
+    * An issue where subsequent updates were locked out after the first one completes.
+      This happened when custom download transport was not used.
+
+* Renamed the nRF Cloud cellular positioning library to :ref:`lib_nrf_cloud_location`.
+  In addition to cellular location, the library now supports device location from nRF Cloud using Wi-Fi network information.
+
+Libraries for NFC
+-----------------
+
+* :ref:`lib_nfc_ndef`:
+
+  * Fixed a write to a constant field in the :c:func:`ac_rec_payload_parse` function.
+
+* :ref:`nfc_t4t_isodep_readme`:
+
+  * Fixed out of bounds access in the :c:func:`ats_parse` function.
+
+nRF RPC libraries
+-----------------
+
+* |no_changes_yet_note|
+
+Other libraries
+---------------
+
+* Added:
+
+  * :ref:`lib_sfloat` library.
+  * :ref:`lib_hw_id` library to retrieve a unique hardware ID.
+
+* :ref:`emds_readme` library:
+
+  * Updated the library implementation to bypass the flash driver when storing the emergency data.
+    This allows calling the :c:func:`emds_store` function from an interrupt context.
+
+  * Removed the internal thread for storing the emergency data.
+    The emergency data is now stored by the :c:func:`emds_store` function.
+
+* :ref:`mod_dm` module:
+
+  * Added a window length configuration to be used runtime, when a new measurement request is added.
+
+  * Updated:
+
+    * The calculation of MPSL timeslot length has been improved by using the :ref:`nrfxlib:nrf_dm` functionality.
+    * Renamed the ``access_address`` field to ``rng_seed`` in the :c:struct:`dm_request` structure.
+
+* :ref:`app_event_manager`:
+
+  * Added :kconfig:option:`CONFIG_APP_EVENT_MANAGER_SHELL` Kconfig option.
+    The option can be used to disable Event Manager shell commands.
+
+* :ref:`nrf_profiler`:
+
+  * Added the :kconfig:option:`CONFIG_NRF_PROFILER_SHELL` Kconfig option.
+    The option can be used to disable the nRF Profiler shell commands.
+
+Common Application Framework (CAF)
+----------------------------------
+
+* :ref:`caf_ble_state`:
+
+  * Added :kconfig:option:`CONFIG_CAF_BLE_STATE_MAX_LOCAL_ID_BONDS`.
+    The option allows to specify the maximum number of allowed bonds per Bluetooth local identity for a Bluetooth Peripheral.
```

Comment:
```suggestion
    The option allows to specify the maximum number of allowed bonds in each Bluetooth local identity for a Bluetooth Peripheral.
```
We are supposed to avoid the Latin-based words like "per", "via" and so forth, right?

Reply:
Yes. Fixed.